1.1/reqs.ttl

```diff
@@ -0,0 +1,951 @@
+             skos:prefLabel "GeoSPARQL 1.0 Conformance Class: Core"@en .
+
+confs10:geometry-extension rdf:type owl:NamedIndividual ,
+                                    skos:Concept ;
+                           skos:member conf10gx:feature-properties ,
+                                       conf10gx:geometry-as-gml-literal ,
+                                       conf10gx:geometry-as-wkt-literal ,
+                                       conf10gx:geometry-properties ,
+                                       conf10gx:gml-literal ,
+                                       conf10gx:gml-literal-empty ,
+                                       conf10gx:gml-profile ,
+                                       conf10gx:query-functions ,
+                                       conf10gx:srid-function ,
+                                       conf10gx:wkt-axis-order ,
+                                       conf10gx:wkt-literal ,
+                                       conf10gx:wkt-literal-default-srs ,
+                                       conf10gx:wkt-literal-empty ;
+                           skos:prefLabel "GeoSPARQL 1.0 Conformance Class: Geometry Extension"@en .
+
+confs10:geometry-topology-extension rdf:type owl:NamedIndividual ,
+                                             skos:Collection ;
+                                    skos:member conf10gtx:eh-query-functions ,
+                                                conf10gtx:rcc8-query-functions ,
+                                                conf10gtx:relate-query-function ,
+                                                conf10gtx:sf-query-functions ;
+                                    skos:prefLabel "GeoSPARQL 1.0 Conformance Class: Geometry Topology Extension"@en .
+
+confs10:rdfs-entailment-extension rdf:type owl:NamedIndividual ,
+                                           skos:Collection ;
+                                  skos:member conf10qre:eh-query-rewrite ,
+                                              conf10qre:rcc8-query-rewrite ,
+                                              conf10qre:sf-query-rewrite ;
+                                  skos:prefLabel "GeoSPARQL 1.0 Conformance Class: RDFS Entailment Extension"@en .
+
+confs10:topology-vocab-extension rdf:type owl:NamedIndividual ,
+                                          skos:Concept ;
+                                 skos:member conf10gtx:eh-query-functions ,
+                                             conf10gtx:rcc8-query-functions ,
+                                             conf10gtx:relate-query-function ,
+                                             conf10gtx:sf-query-functions ;
+                                 skos:prefLabel "GeoSPARQL 1.0 Conformance Class: Topology Vocabulary Extension"@en .
+
+conf10core:feature-class rdf:type owl:NamedIndividual ,
+                                  skos:Concept ;
+                         geo:testPurpose "check conformance with this requirement" ;

```

Comment:
(These comments apply to all similar occurences)

1. The phrase "this requirement" is unclear. How about "Check conformance with the requirement referred to by the subject of the rdfs:seeAlso property of this resource", or something similar?
2.` geo:testPurpose` seems undefined. And the GeoSPARQL ontology does not seem the right place to define such things. It would be better to use a general OGC vocabulary for that. That goes for `geo:testType` and similar terms too.
3. If a more general vocabulary for conformance testing would be made, probably a better way than rdfs:seeAlso can be used to link a conformance test to a requirement.

Perhaps we need a new issue to deal with the necessity of a conformance testing vocabulary at the OGC level.

Reply:
Yes, you are correct, there are vocabularies that need to be defined. I think the OGC Naming Authority will do this?
And yes I agree that "this requirement" is not very precise.
However, it is exactly the text that is written in Annex A. So either we decide to copy this text 1:1 or we use a different description. For now, I used the exact definitions from Annex A.

Reply:
> Yes, you are correct, there are vocabularies that need to be defined. I think the OGC Naming Authority will do this?

The Naming Authority does come to mind. Maybe we can cooperate - propose a vocabulary for the NA to publish? @nicholascar will probably have an idea about a good approach. 

> And yes I agree that "this requirement" is not very precise.
> However, it is exactly the text that is written in Annex A. So either we decide to copy this text 1:1 or we use a different description. For now, I used the exact definitions from Annex A.

When viewed as text the relationship is much clearer (¨requirement¨ printed in boldface). It will probably help to have a vocabulary for requirements/conformance classes. It could contain a property to relate a conformance test to a requirement.

Reply:
Correct. For now, this is only achieved by the rdfs:seeAlso statement which might be replaced by another vocabulary once we define one